errbot-reminder.py

```diff
@@ -0,0 +1,101 @@
+from datetime import date, datetime, time, timedelta
+
+from errbot import BotPlugin, botcmd
+
+# Daily / Retrospective --> If it's an ordinary day or a retrospective, meeting at 9:30
+# Sprint Planning --> If it's a Sprint planning, meeting at 15:30
+# Sprint review --> If it's a Sprint Review, meeting at 14:45
+#  -----------------
+
+
+class Reminder(BotPlugin):
+    @botcmd
+    def whereami(self, msg, args):
+        stream = msg._from._room._id
+        topic = msg._from._room._subject
+        # return f"You are in #{stream}>{topic}"
```

Comment:
could you please remove the comments inside the class?

Reply:
It's done